src/devtools.ts

```diff
@@ -71,4 +69,7 @@ customGlobal.window.__REACT_DEVTOOLS_COMPONENT_FILTERS__ = [
 	}
 ];
 
+// @ts-ignore
+import {connectToDevTools} from 'react-devtools-core';
```

Comment:
This will not work when we eventually switch to transpile to native ESM, as `import`'s are hoisted to the top. Just something to keep in mind.

I think we could move the `window` polyfilling to a separate file and import it before this import.

Reply:
Yep, if I set `"module": "es2015"` in ink's `tsconfig.json` and run my project with `-r esm` it does not work. I move `window `polyfilling to a separate file